apps/ledger-live-mobile/src/config/urls.tsx

```diff
@@ -153,6 +153,9 @@ export const urls = {
     github: "https://github.com/LedgerHQ",
     status: "https://status.ledger.com",
   },
+  deviceLocalization: {
+    learnMore: "https://support.ledger.com/hc/en-us/articles/6522080681501",
```

Comment:
URL is inaccessible but I assume it will be made available eventually

Reply:
Indeed. The article is not published yet but it will be this URL